data/transactions/logic/opcodes.go

```diff
@@ -28,6 +28,11 @@ const LogicVersion = 2
 // be maliciously or accidentally rekeyed.
 const rekeyingEnabledVersion = 2
 
+// appsEnabledVersion is the version of TEAL where ApplicationCall
+// functionality was enabled. We use this to disallow v0 and v1 TEAL programs
+// from being used with applications.
+const appsEnabledVersion = 2
```

Comment:
maybe here should be a comment like "do not edit" or a test that ensures `appsEnabledVersion == 2` to prevent accidental constants change.

Reply:
Added a "do not edit" comment